lib/MediaWords/DBI/Stories/AP.pm

```diff
@@ -114,6 +133,58 @@ SQL
     return $sentence_lengths;
 }
 
+# given raw html content, extract the content and parse it into sentences using the existing mediacloud pipeline code
+sub _get_sentences_from_content($)
```

Comment:
From a purist's POV, we should probably reduce the number of "extract HTML to sentences" methods into a single subroutine that we use everywhere (right now we have ~4 implementations).

Reply:
I agree in theory, but I'm not going to jump into that mess for this change.